### ISSUE TITLE
Pi4 & RaspBee 2: Can't connect light or accessories to deCONZ Docker

### ISSUE THREAD

**Issue by mr-bluescreen** [marthoc/docker-deconz#247]
Hi,

I've switched from an Raspberry Pi 3 B+ to the new Raspberry Pi 4 with 8 GB of RAM and am now trying to migrate my deCONZ Setup to the new Pi.

I'm using the RaspBee II on latest Rasbian.

I followed every step from the Docker installation guide and can connect to the deCONZ Web UI but can't connect any devices, light or accessories via ZigBee. Nothing is found :-(
Configuring the RaspBee module on the old 3B+ without docker and everything is working as expected (added Hue Light, Aqara Sensor and Hue Dimmer Switch with no problem)

The RTC is running fine:
```shell
hwclock from util-linux 2.33.1
System Time: 1596325793.077762
Trying to open: /dev/rtc0
Using the rtc interface to the clock.
Last drift adjustment done at 1596197280 seconds after 1969
Last calibration done at 1596197280 seconds after 1969
Hardware clock is on UTC time
Assuming hardware clock is kept in UTC time.
Waiting for clock tick...
ioctl(3, RTC_UIE_ON, 0): Invalid argument
Waiting in loop for time from /dev/rtc0 to change
...got clock tick
Time read from Hardware Clock: 2020/08/01 23:49:55
Hw clock time : 2020/08/01 23:49:55 = 1596325795 seconds since 1969
Time since last adjustment is 128515 seconds
Calculated Hardware Clock drift is 0.000000 seconds
2020-08-02 01:49:54.069732+02:00
```

Here are some details from the WebUI:

```
Produkt: RaspBee
Version: 2.05.79 / 22.5.2020
Firmware: 26610700
ZigBee Kanal: 15
Netzwerk ID: CA03
```

And this is my docker-compose.yml:

```yaml
version: '2'

services:
  deconz:
    image: marthoc/deconz
    container_name: deconz
    network_mode: host
    devices:
      - /dev/ttyAMA0
    environment:
      - DECONZ_WEB_PORT=80
      - DECONZ_WS_PORT=443
      - DEBUG_INFO=1
      - DEBUG_APS=0
      - DEBUG_ZCL=0
      - DEBUG_ZDP=0
      - DEBUG_OTAU=0
      - TZ=Europe/Berlin
    volumes:
      - /mnt/usb/docker_compose_data/deCONZ:/root/.local/share/dresden-elektronik/deCONZ
    restart: always
```

Thank you for your help, I hope somone could help me!

**Comment by mr-bluescreen** [comment]
More testing:

When installed directly on the Pi 4, deCONZ is working fine. I can adopt all lights and sensors in my home and controll them.
(-> so there can't be a problem concerning the serial interface options or rights?!)
Runnig the same Raspbee II on the same Pi 4 in Docker, deCONZ won' find any ZigBee device.

I've crated a deCONZ backup while runnig directly on the Pi 4 and uploaded it to the deCONZ software running in Docker.

Here is where things get strange:
- Every adopted devices shows up.
- Lights aren't controllable (Tradfri and Hue lamps)
- Hue Dimmer switch isn't responding
BUT
- My cheap Aqara Door Window sensor is showing it's open/close staus as expected

In conclusion the serial interface /dev/ttyAMA0 seems to be mapped correctly to the Docker container and is the right one (Docker deCONZ knows Firmware and ZigBee details and Aqara sensor ist working)

So something else is going wrong here...

**Comment by mr-bluescreen** [comment]
I've turned on every debug option for the docker container and turned on one single light.

This is the output (which I cannot do much with)

````
15:47:28:484 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:28:484 APS-DATA.indication request id: 225 -> finished
15:47:28:484 APS-DATA.request id: 225 erase from queue
15:47:28:484 ZDP status = 0x00 -> SUCCESS
15:47:28:484 ZDP Mgmt_Lqi_rsp zdpSeq: 55 from 0x00212EFFFF05BD9B total: 5, startIndex: 4, listCount: 1
15:47:28:485     * neighbor: 0x00158D00045CF07F (0x7E87), LQI: 255, relation: 0x01 rxOnWHenIdle: 0
15:47:28:892 don't close database yet, keep open for 900 seconds
15:47:28:953 dev /dev/ttyAMA0
15:47:28:954 APS-DATA.request id: 229, addrmode: 0x03, addr: 0x0017880103028268, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options 0x00
15:47:28:954 GW firmware version: 0x26610700
15:47:29:373 APS-DATA.request id: 232, addrmode: 0x02, addr: 0xfffd, profile: 0x0000, cluster: 0x0000, ep: 0x00 -> 0x00 queue: 1 len: 11 tx.options 0x00
15:47:29:374 send NWK_addr_req to 0x90FD9FFFFED30A7F, last seen 0 s, last seen by neighbors 0 s
15:47:29:557 APS-DATA.confirm id: 232, status: 0xE1
15:47:30:062 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task
15:47:30:062 APS-DATA.confirm id: 229, status: 0xE1
15:47:31:642 APS-DATA.request id: 241, addrmode: 0x03, addr: 0x90fd9ffffe4b5ccd, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x01 queue: 0 len: 5 tx.options 0x00
15:47:32:733 0x90FD9FFFFE4B5CCD error APSDE-DATA.confirm: 0xE1 on task
15:47:32:733 APS-DATA.confirm id: 241, status: 0xE1
15:47:32:734 Mgmt_Lqi_req zdpSeq: 57 to 0x90FD9FFFFED30A7F start index 0
15:47:32:734 APS-DATA.request id: 247, addrmode: 0x03, addr: 0x90fd9ffffed30a7f, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
15:47:33:442 APS-DATA.request id: 251, addrmode: 0x03, addr: 0x001788010320e3b6, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 1 len: 5 tx.options 0x00
15:47:33:839 APS-DATA.confirm id: 247, status: 0xE1
15:47:34:558 0x001788010320E3B6 error APSDE-DATA.confirm: 0xE1 on task
15:47:34:558 APS-DATA.confirm id: 251, status: 0xE1
15:47:35:242 APS-DATA.request id: 3, addrmode: 0x03, addr: 0x0017880103028268, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options
0x00
15:47:36:094 Mgmt_Lqi_req zdpSeq: 58 to 0x00212EFFFF05BD9B start index 0
15:47:36:094 APS-DATA.request id: 8, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options
0x00
15:47:36:136 APS-DATA.confirm id: 8, status: 0x00 SUCCESS
15:47:36:136 APS-DATA.confirm request id: 8 -> confirmed, timeout 28797880
15:47:36:168 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:36:168 APS-DATA.indication request id: 8 -> finished
15:47:36:168 APS-DATA.request id: 8 erase from queue
15:47:36:168 ZDP status = 0x00 -> SUCCESS
15:47:36:168 ZDP Mgmt_Lqi_rsp zdpSeq: 58 from 0x00212EFFFF05BD9B total: 5, startIndex: 0, listCount: 2
15:47:36:168     * neighbor: 0xD0CF5EFFFEC22D19 (0xBD67), LQI: 221, relation: 0x02 rxOnWHenIdle: 1
15:47:36:168     * neighbor: 0x0017880103028268 (0x2167), LQI: 249, relation: 0x02 rxOnWHenIdle: 1
15:47:36:351 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task
15:47:36:351 APS-DATA.confirm id: 3, status: 0xE1
15:47:37:842 APS-DATA.request id: 16, addrmode: 0x03, addr: 0x90fd9ffffe4b5ccd, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x01 queue: 0 len: 5 tx.options 0x00
15:47:38:494 Mgmt_Lqi_req zdpSeq: 59 to 0x00212EFFFF05BD9B start index 2
15:47:38:494 APS-DATA.request id: 20, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:47:38:537 APS-DATA.confirm id: 20, status: 0x00 SUCCESS
15:47:38:537 APS-DATA.confirm request id: 20 -> confirmed, timeout 28610984
15:47:38:568 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:38:568 APS-DATA.indication request id: 20 -> finished
15:47:38:568 APS-DATA.request id: 20 erase from queue
15:47:38:568 ZDP status = 0x00 -> SUCCESS
15:47:38:568 ZDP Mgmt_Lqi_rsp zdpSeq: 59 from 0x00212EFFFF05BD9B total: 5, startIndex: 2, listCount: 2
15:47:38:568     * neighbor: 0x90FD9FFFFE4B5CCD (0x0A40), LQI: 255, relation: 0x02 rxOnWHenIdle: 1
15:47:38:568     * neighbor: 0x90FD9FFFFED30A7F (0xB73F), LQI: 243, relation: 0x02 rxOnWHenIdle: 1
15:47:38:908 dev /dev/ttyAMA0
15:47:38:908 GW firmware version: 0x26610700
15:47:38:967 0x90FD9FFFFE4B5CCD error APSDE-DATA.confirm: 0xE1 on task
15:47:38:967 APS-DATA.confirm id: 16, status: 0xE1
15:47:39:642 APS-DATA.request id: 27, addrmode: 0x03, addr: 0x001788010320e3b6, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options 0x00
15:47:40:517 APS-DATA.request id: 32, addrmode: 0x03, addr: 0x90fd9ffffe4b5ccd, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x01 queue: 1 len: 3 tx.options 0x04
15:47:40:740 0x001788010320E3B6 error APSDE-DATA.confirm: 0xE1 on task
15:47:40:740 APS-DATA.confirm id: 27, status: 0xE1
15:47:40:894 Mgmt_Lqi_req zdpSeq: 60 to 0x00212EFFFF05BD9B start index 4
15:47:40:894 APS-DATA.request id: 35, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:47:40:972 don't close database yet, keep open for 900 seconds
15:47:40:984 APS-DATA.confirm id: 35, status: 0x00 SUCCESS
15:47:40:984 APS-DATA.confirm request id: 35 -> confirmed, timeout 28302320
15:47:41:009 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:41:009 APS-DATA.indication request id: 35 -> finished
15:47:41:009 APS-DATA.request id: 35 erase from queue
15:47:41:009 ZDP status = 0x00 -> SUCCESS
15:47:41:009 ZDP Mgmt_Lqi_rsp zdpSeq: 60 from 0x00212EFFFF05BD9B total: 5, startIndex: 4, listCount: 1
15:47:41:009     * neighbor: 0x00158D00045CF07F (0x7E87), LQI: 255, relation: 0x01 rxOnWHenIdle: 0
15:47:41:442 APS-DATA.request id: 39, addrmode: 0x03, addr: 0x0017880103028268, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 1 len: 5 tx.options 0x00
15:47:42:520 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task
15:47:42:520 APS-DATA.confirm id: 39, status: 0xE1
15:47:44:142 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:242 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:342 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:442 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:542 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:642 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:734 Mgmt_Lqi_req zdpSeq: 61 to 0xD0CF5EFFFEC22D19 start index 0
15:47:44:734 APS-DATA.request id: 54, addrmode: 0x03, addr: 0xd0cf5efffec22d19, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:47:44:742 delay sending request 50 dt 4 ms to 0x90FD9FFFFE4B5CCD, ep: 0x01 cluster: 0x0006 onAir: 1
15:47:44:842 APS-DATA.request id: 50, addrmode: 0x03, addr: 0x90fd9ffffe4b5ccd, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x01 queue: 2 len: 5 tx.options 0x00
15:47:45:776 APS-DATA.confirm id: 54, status: 0xE1
15:47:45:898 don't close database yet, keep open for 900 seconds
15:47:45:909 0x90FD9FFFFE4B5CCD error APSDE-DATA.confirm: 0xE1 on task
15:47:45:909 APS-DATA.confirm id: 50, status: 0xE1
15:47:46:642 APS-DATA.request id: 62, addrmode: 0x03, addr: 0x001788010320e3b6, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 1 len: 5 tx.options 0x00
15:47:47:728 0x001788010320E3B6 error APSDE-DATA.confirm: 0xE1 on task
15:47:47:728 APS-DATA.confirm id: 62, status: 0xE1
15:47:48:442 APS-DATA.request id: 71, addrmode: 0x03, addr: 0x0017880103028268, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 1 len: 5 tx.options 0x00
15:47:48:574 Mgmt_Lqi_req zdpSeq: 62 to 0x00212EFFFF05BD9B start index 0
15:47:48:574 APS-DATA.request id: 72, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 2 len: 2 tx.options 0x00
15:47:48:617 APS-DATA.confirm id: 72, status: 0x00 SUCCESS
15:47:48:617 APS-DATA.confirm request id: 72 -> confirmed, timeout 28268360
15:47:48:649 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:48:649 APS-DATA.indication request id: 72 -> finished
15:47:48:649 APS-DATA.request id: 72 erase from queue
15:47:48:649 ZDP status = 0x00 -> SUCCESS
15:47:48:649 ZDP Mgmt_Lqi_rsp zdpSeq: 62 from 0x00212EFFFF05BD9B total: 5, startIndex: 0, listCount: 2
15:47:48:649     * neighbor: 0xD0CF5EFFFEC22D19 (0xBD67), LQI: 223, relation: 0x02 rxOnWHenIdle: 1
15:47:48:649     * neighbor: 0x0017880103028268 (0x2167), LQI: 249, relation: 0x02 rxOnWHenIdle: 1
15:47:48:916 dev /dev/ttyAMA0
15:47:48:917 GW firmware version: 0x26610700
15:47:49:563 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task
15:47:49:563 APS-DATA.confirm id: 71, status: 0xE1
15:47:50:888 Current channel 15
15:47:50:909 Device TTL 2282 s flags: 0x7
15:47:50:974 Mgmt_Lqi_req zdpSeq: 63 to 0x00212EFFFF05BD9B start index 2
15:47:50:974 APS-DATA.request id: 84, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:47:51:019 APS-DATA.confirm id: 84, status: 0x00 SUCCESS
15:47:51:019 APS-DATA.confirm request id: 84 -> confirmed, timeout 28268360
15:47:51:050 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:51:050 APS-DATA.indication request id: 84 -> finished
15:47:51:050 APS-DATA.request id: 84 erase from queue
15:47:51:050 ZDP status = 0x00 -> SUCCESS
15:47:51:050 ZDP Mgmt_Lqi_rsp zdpSeq: 63 from 0x00212EFFFF05BD9B total: 5, startIndex: 2, listCount: 2
15:47:51:050     * neighbor: 0x90FD9FFFFE4B5CCD (0x0A40), LQI: 255, relation: 0x02 rxOnWHenIdle: 1
15:47:51:050     * neighbor: 0x90FD9FFFFED30A7F (0xB73F), LQI: 243, relation: 0x02 rxOnWHenIdle: 1
15:47:51:142 APS-DATA.request id: 86, addrmode: 0x03, addr: 0x90fd9ffffe4b5ccd, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x01 queue: 1 len: 5 tx.options 0x00
15:47:51:641 0x90FD9FFFFE4B5CCD error APSDE-DATA.confirm: 0xD0 on task
15:47:51:641 APS-DATA.confirm id: 32, status: 0xD0
15:47:51:641 max transmit errors for node 0x90FD9FFFFE4B5CCD, last seen by neighbors 0 s
15:47:52:241 0x90FD9FFFFE4B5CCD error APSDE-DATA.confirm: 0xE1 on task
15:47:52:241 APS-DATA.confirm id: 86, status: 0xE1
15:47:52:902 APS-DATA.request id: 96, addrmode: 0x02, addr: 0xfffd, profile: 0x0000, cluster: 0x0000, ep: 0x00 -> 0x00 queue: 0 len: 11 tx.options 0x00
15:47:52:902 send NWK_addr_req to 0x001788010320E3B6, last seen 0 s, last seen by neighbors 0 s
15:47:52:942 APS-DATA.request id: 95, addrmode: 0x03, addr: 0x001788010320e3b6, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 1 len: 5 tx.options 0x00
15:47:53:126 APS-DATA.confirm id: 96, status: 0xE1
15:47:53:134 0x0A40 seems to be a zombie recv errors 10
15:47:53:134 LightNode removed 0x90fd9ffffe4b5ccd
15:47:53:134 Node zombie state changed 0x90fd9ffffe4b5ccd
15:47:53:374 Mgmt_Lqi_req zdpSeq: 65 to 0x00212EFFFF05BD9B start index 4
15:47:53:374 APS-DATA.request id: 99, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:47:53:416 APS-DATA.confirm id: 99, status: 0x00 SUCCESS
15:47:53:416 APS-DATA.confirm request id: 99 -> confirmed, timeout 29765256
15:47:53:442 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 4, rssi: 0
15:47:53:442 APS-DATA.indication request id: 99 -> finished
15:47:53:442 APS-DATA.request id: 99 erase from queue
15:47:53:442 ZDP status = 0x00 -> SUCCESS
15:47:53:442 ZDP Mgmt_Lqi_rsp zdpSeq: 65 from 0x00212EFFFF05BD9B total: 5, startIndex: 4, listCount: 1
15:47:53:442     * neighbor: 0x00158D00045CF07F (0x7E87), LQI: 255, relation: 0x01 rxOnWHenIdle: 0
15:47:55:887 don't close database yet, keep open for 900 seconds
15:47:55:900 0x001788010320E3B6 error APSDE-DATA.confirm: 0xE1 on task
15:47:55:900 APS-DATA.confirm id: 95, status: 0xE1
15:47:56:629 APS-DATA.request id: 106, addrmode: 0x03, addr: 0x0017880103028268, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options 0x00
15:47:57:742 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task
15:47:57:742 APS-DATA.confirm id: 106, status: 0xE1
15:47:58:911 dev /dev/ttyAMA0
15:47:58:912 GW firmware version: 0x26610700
15:47:59:254 Mgmt_Lqi_req zdpSeq: 66 to 0x00212EFFFF05BD9B start index 0
15:47:59:254 APS-DATA.request id: 118, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
15:47:59:297 APS-DATA.confirm id: 118, status: 0x00 SUCCESS
15:47:59:297 APS-DATA.confirm request id: 118 -> confirmed, timeout 29647576
15:47:59:328 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:47:59:328 APS-DATA.indication request id: 118 -> finished
15:47:59:328 APS-DATA.request id: 118 erase from queue
15:47:59:328 ZDP status = 0x00 -> SUCCESS
15:47:59:328 ZDP Mgmt_Lqi_rsp zdpSeq: 66 from 0x00212EFFFF05BD9B total: 5, startIndex: 0, listCount: 2
15:47:59:328     * neighbor: 0xD0CF5EFFFEC22D19 (0xBD67), LQI: 223, relation: 0x02 rxOnWHenIdle: 1
15:47:59:328     * neighbor: 0x0017880103028268 (0x2167), LQI: 249, relation: 0x02 rxOnWHenIdle: 1
15:47:59:342 APS-DATA.request id: 119, addrmode: 0x03, addr: 0x001788010320e3b6, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options 0x00
15:48:00:431 0x001788010320E3B6 error APSDE-DATA.confirm: 0xE1 on task
15:48:00:431 APS-DATA.confirm id: 119, status: 0xE1
15:48:01:141 APS-DATA.request id: 128, addrmode: 0x03, addr: 0x0017880103028268, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options 0x00
15:48:01:654 Mgmt_Lqi_req zdpSeq: 67 to 0x00212EFFFF05BD9B start index 2
15:48:01:654 APS-DATA.request id: 132, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:48:01:702 APS-DATA.confirm id: 132, status: 0x00 SUCCESS
15:48:01:702 APS-DATA.confirm request id: 132 -> confirmed, timeout 28143880
15:48:01:733 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 1, rssi: -11
15:48:01:733 APS-DATA.indication request id: 132 -> finished
15:48:01:733 APS-DATA.request id: 132 erase from queue
15:48:01:733 ZDP status = 0x00 -> SUCCESS
15:48:01:733 ZDP Mgmt_Lqi_rsp zdpSeq: 67 from 0x00212EFFFF05BD9B total: 5, startIndex: 2, listCount: 2
15:48:01:733     * neighbor: 0x90FD9FFFFE4B5CCD (0x0A40), LQI: 255, relation: 0x02 rxOnWHenIdle: 1
15:48:01:733     * neighbor: 0x90FD9FFFFED30A7F (0xB73F), LQI: 242, relation: 0x02 rxOnWHenIdle: 1
15:48:02:134 discovery for zombie 0x90fd9ffffe4b5ccd dropped, last try was 238 s ago
15:48:02:254 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task
15:48:02:254 APS-DATA.confirm id: 128, status: 0xE1
15:48:03:842 APS-DATA.request id: 142, addrmode: 0x03, addr: 0x001788010320e3b6, profile: 0x0104, cluster: 0x0006, ep: 0x01 -> 0x0B queue: 0 len: 5 tx.options 0x00
15:48:04:054 Mgmt_Lqi_req zdpSeq: 68 to 0x00212EFFFF05BD9B start index 4
15:48:04:054 APS-DATA.request id: 144, addrmode: 0x03, addr: 0x00212effff05bd9b, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 1 len: 2 tx.options 0x00
15:48:04:121 don't close database yet, keep open for 900 seconds
15:48:04:133 APS-DATA.confirm id: 144, status: 0x00 SUCCESS
15:48:04:133 APS-DATA.confirm request id: 144 -> confirmed, timeout 28268360
15:48:04:159 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 17, rssi: 21
15:48:04:159 APS-DATA.indication request id: 144 -> finished
15:48:04:159 APS-DATA.request id: 144 erase from queue
15:48:04:159 ZDP status = 0x00 -> SUCCESS
````

**Comment by mr-bluescreen** [comment]
Finally! I got the solution by myself!

For everyone who is experiencing simmilar errors: **!!! Don't use USB 3 on the Raspberry Pi 4 !!!** 

I searched for the error `15:48:02:254 0x0017880103028268 error APSDE-DATA.confirm: 0xE1 on task` from my logfile above and came to this site:
https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Zigbee-Error-Codes-in-the-Log

Where it says, that using USB 3.0 near a ConBee II stick has an dramaitc effect on wireless performance. Apparently this is also true for RasBee HATs.
I'm using an external drive (see the docker-compose.yaml > /mnt/usb/) to store the persistent Docker Volumes. I now switched to USB 2.0 and all problems are gone!
